build-system/amp.extern.js

```diff
@@ -35,7 +35,23 @@ window.AMP = {};
 
 // Externed explicitly because we do not export Class shaped names
 // by default.
-window.AMP.BaseElement;
+/**
+ * @constructor
+ */
+window.AMP.BaseElement = function(element) {};
+
+window.AMP.BaseElement.prototype.layout_;
```

Comment:
this will prevent closure compiler from renaming the properties. right now theres a chance that these properties if renamed will have a collision with a renamed property in the extension (symbol reuse)